example_test.go

```diff
@@ -131,6 +131,10 @@ func ExampleJob_Weekday() {
 	// Output:
 	// Monday
 	// job not scheduled weekly on a weekday
+
+	secondWeeklyJob, _ := s.Every(1).Week().Monday().Friday().Do(task)
+	weeldays, _ := secondWeeklyJob.Weekday()
```

Comment:
```suggestion
	weekdays, _ := secondWeeklyJob.Weekday()
```

Reply:
the test that involves this typo will be moved to another function